src/components/Section/Section.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import classnames from 'classnames'
+import styles from './Section.module.scss'
+
+const Section = ({ className, children }) => (
+  <section className={classnames(styles.section, className)}>
```

Comment:
Add the section className as well for better bulma semantics (<section className={classnames("section", styles.section, className}>

Reply:
fixed